src/tests/ftest/io/nvme_io.py

```diff
@@ -119,13 +95,14 @@ def test_nvme_io(self):
         sizes.IOR will be run for all Object type supported. Purpose is to
         verify pool size (SCM and NVMe) for IOR file.
         This test is running multiple IOR on same server start instance.
-        :avocado: tags=nvme,nvme_io,large
+        :avocado: tags=hw,nvme,large
         """
         ior_args = {}
 
         hostlist_clients = self.params.get("clients", '/run/hosts/*')
```

Comment:
Is this different than the self.hostlist_clients provided by the inherited TestWithServers class?